src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,49 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAXIMUM_CAPACITY = 10;
+    private K[] keyArray;
+    private V[] valueArray;
+
+    public StorageImpl() {
+        keyArray = (K[]) new Object[MAXIMUM_CAPACITY];
+        valueArray = (V[]) new Object[MAXIMUM_CAPACITY];
+    }
+
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < MAXIMUM_CAPACITY; i++) {
+            if ((keyArray[i] == null && valueArray[i] == null)
```

Comment:
```suggestion
            if ((keyArray[i] == valueArray[i])
```
good way to compare nulls